### ISSUE TITLE
[Feature Request]: Request to have function requestable asset with date column

### ISSUE THREAD

**Issue by Asyraf777** [grokability/snipe-it#11683]
### Is your feature request related to a problem? Please describe.

The request feature currently just can request item, it is more realistic if got the date functionality for this issue 

### Describe the solution you'd like

Add date to use in the request section

**Comment by brianhoganm** [comment]
I totally agree. I am using SnipeIT in production mode for a large school district and having the option to select a date would be very beneficial.

**Comment by PonlorkBeang** [comment]
Requested item still available in "Requestable Assets"

I see a PC requested by 1 user and it still available for other request.
After I accepted to testing user i see Check In Status as below picture. 
![image](https://user-images.githubusercontent.com/97224078/187132983-7e67ddf2-30fa-48b0-a6b7-f40edf89e2c1.png)
 it should show the available PC (not request) only.

**Comment by CloCkWeRX** [comment]
Duplicate of/worth merging with #9435 ?
